web/app/package.json

```diff
@@ -6,6 +6,7 @@
   "dependencies": {
     "@material-ui/core": "3.6.1",
     "@material-ui/icons": "3.0.1",
+    "@wdio/sauce-service": "^5.7.15",
```

Comment:
This should be moved into `devDependencies` rather than here since we don't need to run this module production. Also, it looks like we only use this module for testing so `devDependencies` is a more appropriate place to have this.

Reply:
Alright! will change that.
Unfortunately currently going through a stressful week in college [finals and project submissions begin from next week :) ]. So I'll try to do the required changes(including the ones in the aforementioned comments) whenever I can get the time. So sorry about that. 
Thank you for reviewing my PR!